remote-api/src/main/resources/alfresco/public-rest-context.xml

```diff
@@ -910,6 +910,9 @@
         <constructor-arg name="actionService" ref="ActionService"/>
         <constructor-arg name="dictionaryService" ref="DictionaryService"/>
         <constructor-arg name="namespaceService" ref="NamespaceService"/>
+        <constructor-arg name="nodeService" ref="NodeService"/>
```

Comment:
This one may be obsolete (nod used in the Java class).

Reply:
Thanks - good spot.